tensorflow/compiler/mlir/xla/tests/hlo-legalize-to-lhlo.mlir

```diff
@@ -13,42 +13,51 @@ func @attrs_copy(%operand: memref<2x2xf32>, %result: memref<2x2xf32>) {
 
 // CHECK-LABEL: func @func_op
```

Comment:
(transmitting @joker-eph): Why isn't the copy removal tested independently of the legalization? Isn't this an optimization?

In general we try to keep testing a minimal granularity: I would rather see `-lhlo-redundant-copies-removal ` being tested independently on a LHLO input test file.